src/main/java/com/blackducksoftware/integration/hub/api/component/version/ComplexLicenseItem.java

```diff
@@ -25,9 +25,6 @@
 
 import java.util.List;
 
-import org.apache.commons.lang3.builder.RecursiveToStringStyle;
-import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
-
 import com.blackducksoftware.integration.hub.api.item.HubItem;
 
 public class ComplexLicenseItem extends HubItem {
```

Comment:
the getter for licenses should probably just be called 'getLicenses' - in a world where these model classes are auto-created, the getters would all be named directly after the property they get